src/ember-app.js

```diff
@@ -450,6 +450,8 @@ function createShoebox(doc, fastbootInfo) {
 
     let value = shoebox[key];
     let textValue = JSON.stringify(value);
+    if (textValue === undefined) { continue; }
```

Comment:
Im not sure always logging makes sense. Seems like it could be valid to be undefined?

If it is exceptional then sure, otherwise at most a development only log, but I'm inclined to not over-log.